rfcs/text/0000-create-file-elf.md

```diff
@@ -0,0 +1,160 @@
+# 0000: Create the ELF sub-field of the File fieldset
+
+- Stage: **1 (proposal)**
```

Comment:
```suggestion
- Stage: **1 (draft)**
```

With the changes to the [stages](https://elastic.github.io/ecs/stages.html), stage 1 changed from `proposal` to `draft`.